exchanges/okx/okx_test.go

```diff
@@ -1847,6 +1847,35 @@ func TestUpdateTradablePairs(t *testing.T) {
 	}
 }
 
+func TestUpdateOrderExecutionLimits(t *testing.T) {
+	t.Parallel()
+	if err := ok.UpdateOrderExecutionLimits(context.Background(), asset.Spot); err != nil {
```

Comment:
could we please load all assets

```go
	assets := ok.GetAssetTypes(false)
	for x := range assets {
		if err := ok.UpdateOrderExecutionLimits(context.Background(), assets[x]); err != nil {
			t.Error("Okx UpdateOrderExecutionLimits() error", err)
		}
	}
```

and then check each one. Just so if the API changes in the future for one asset this might let us know where the issue is.

Reply:
Agreed. I was a bit lazy with the reviews.

Reply:
That's done. ✅

Reply:
Actually, _and then check each one_.

Okay, yeah, can do.